oop-exercise/src/Main.java

```diff
@@ -0,0 +1,149 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+interface Shape {
+    double calculateArea();
+
+    double calculatePerimeter();
+
+    void displayProperties();
+}
+
+class Square implements Shape {
```

Comment:
Refactor classes and interfaces into separate files. Only keep the main program in the main file.

Reply:
Sure ayye